packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -72,5 +72,11 @@ export const messages = {
   force_lightning_event_create_text: 'SFDX: Create Lightning Event',
   force_lightning_interface_create_text: 'SFDX: Create Lightning Interface',
   force_source_status_local_text: 'SFDX: View Local Changes',
-  force_source_status_remote_text: 'SFDX: View Changes in Default Scratch Org'
+  force_source_status_remote_text: 'SFDX: View Changes in Default Scratch Org',
+  warning_prompt_file_overwrite:
+    'File(s) with specified path already exists in workspace. Would you like to overwrite?',
```

Comment:
'One or more files with the specified path already exist in your workspace. Do you want to overwrite them?'